### PR TITLE
Fix incorrect column type for link original_url

### PR DESCRIPTION
**Pull request by nicholaszuccarelli**
Fix issue https://github.com/juhasev/laravel-ses/issues/22

### CODE REVIEW

File: src/Migrations/2023_06_23_100614_change_original_url_to_text.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    public function up(): void
+    {
+        Schema::table('laravel_ses_email_links', function (Blueprint $table) {
+            $table->text('original_url')->change();
+        });
+    }
+
+    public function down(): void
+    {
+        Schema::table('laravel_ses_sent_emails', function (Blueprint $table) {
```
Comment (leMaur) [inline src/Migrations/2023_06_23_100614_change_original_url_to_text.php]: Hi @nicholaszuccarelli,
can you please fix the table name?

```suggestion
        Schema::table('laravel_ses_email_links', function (Blueprint $table) {
```

**Review by leMaur — CHANGES_REQUESTED** [review]

**Comment by leMaur** [comment]
Fixes #22

**Comment by nicholaszuccarelli** [comment]
Jesus sorry, don't know how I botched that up. Fixed. Please review :)

**Comment by leMaur** [comment]
No worries at all! I'll merge it.
Thanks for the fix!

**Review by leMaur — APPROVED** [review]

**Pull request merged**